### PR TITLE
 Fixed Metrics & Tests for PR-closed-no-merge

### LINKED ISSUE #572: `pull_request_closed_no_merge` metric is broken
**Describe the bug:**
The `pull_request_closed_no_merge` metric returns a 500 internal server error.

**To Reproduce:**
1. Checkout `dev`
2. Run `augur run --disable-housekeeper`
3. Hit `http://localhost:5000/api/unstable/repo/21000/pull-request-closed-no-merge`
4. Observe error

**Expected behavior:**
I expected the metric to return a valid response object, even if it was empty.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS v 10.14.6

**Additional context**
I noticed that there is a missing `GROUP_BY` clause in the SQL query, based on the usage of `COUNT`. This appears to be the main culprit of the issue.

Broken Travis build: https://travis-ci.org/chaoss/augur/builds/657396384

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Fixes #572 

The 404 error is nowhere to be seen now in the metrics test.
All of the tests are passing locally! :D

- Fixed SQL Query
- Added GroupMetric in API docs
- Fixed metric test route

### CODE REVIEW

**Comment by mrsaicharan1** [comment]
Screenshot 2020-03-03 at 2 29 43 AM

File: test/metrics/test_pull_request_metrics.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def test_pull_request_acceptance_rate(metrics):
                                      end_date='2019-12-31 23:59:59',group_by='year').iloc[0]['rate'] > 0
 
 def test_pull_request_closed_no_merge(metrics):
-    assert metrics.pull_requests_closed_no_merge(24).iloc[0]['count'] > 0
-    assert metrics.pull_requests_closed_no_merge(24, 21000, begin_date='2018-1-1 00:00:00',
-                                     end_date='2019-12-31 23:59:59', group_by='year').iloc[0]['count'] > 0
+    assert metrics.pull_requests_closed_no_merge(10).iloc[0]['pr_count'] > 0
+    # assert metrics.pull_requests_closed_no_merge(10, 20, begin_date='2018-1-1 00:00:00',
```
Comment (mrsaicharan1) [inline test/metrics/test_pull_request_metrics.py]: @ccarterlandis I fixed the tests & routes. This is the only case that I'm not able to debug as to why I'm receiving an `index out of bounds` error. Would appreciate it if you could take a look!

Comment (mrsaicharan1) [reply test/metrics/test_pull_request_metrics.py]: Haha never mind! I was using the wrong repo_id to test! It's working perfectly alright 😃

**Comment by mrsaicharan1** [comment]
I also had a question regarding the test repo_ids and repo_group_ids..... Does the Travis deployment have the same test IDs as that as of `repo_load_sample.csv` & `repo_group_load_sample.csv` ?

**Comment by ccarterlandis** [comment]
@mrsaicharan1 for now, for the testing IDs use `24` as the `repo_group_id` and `21000` as the `repo_id` when you commit. We're currently in the process of updating our test data, so use those values until further notice :)

**Comment by mrsaicharan1** [comment]
😃😃 Will follow a test driven development for better results from now on!

**Pull request merged**